test/goalc/source_templates/with_game/test-vector-abs.gc

```diff
@@ -0,0 +1,23 @@
+(defun test-vector-abs ()
+  (let ((vector-0 (new 'stack 'vector))
+        (vector-1 (new 'stack 'vector)))
+		; TODO - a lib function get generate a vector would be useful
```

Comment:
Initialization of stack variables in GOAL is kinda weird and I don't 100% understand when they are/aren't initialized so for now you are stuck with this.  In the near future you will be able to use this syntax for stack variables too:
```
(new 'stack 'vector :x 1.0 :y 2.0 :z 3.0 :w 4.0)
```
but it isn't done yet.

Reply:
ah ok, that's much nicer.  I might still make the function to make writings tests easier _now_ but then it can be easily refactored to the better syntax when available.